### ISSUE TITLE
Comment on last USE statement causes false positive for PSR2.Namespaces.UseDeclaration.SpaceAfterLastUse

### ISSUE THREAD

**Issue by iammattcoleman** [squizlabs/PHP_CodeSniffer#1891]
# Overview
If the last USE statement has a comment, an incorrect `PSR2.Namespaces.UseDeclaration.SpaceAfterLastUse` error is reported.

# Steps to reproduce
Run `phpcs --standard=PSR2` on a file with the following contents:
```php
<?php
use Foo\Bar; //comment

// This should pass.
```

## Expected output
None. This should validate correctly.

## Actual output
```
FILE: /Users/matt/Projects/PHP_CodeSniffer/tests/TestCommentAfterLastUse.php
-------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
-------------------------------------------------------------------------------------
 2 | ERROR | [x] There must be one blank line after the last USE statement; 0 found;
-------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
-------------------------------------------------------------------------------------

Time: 114ms; Memory: 6Mb
```

**Comment by gsherwood** [comment]
> Do you have an ETA for the 3.2.3 release?

Should have been 3 weeks ago but I got very ill, so it's very late. Hopefully this week though.

**Comment by jrfnl** [comment]
@gsherwood Sorry to hear you were ill. Hope you're all better now.
FYI: AFAICS there are four more sniffs with pertinent problems with the new PHPCS annotations. I will try and get to these this week.

**Comment by gsherwood** [comment]
> FYI: AFAICS there are four more sniffs with pertinent problems with the new PHPCS annotations. I will try and get to these this week.

Thanks for letting me know, and thanks for all your help with those annotations.

**Comment by iammattcoleman** [comment]
@gsherwood Thanks for the estimate. Glad you're feeling better!
